base/math.jl

```diff
@@ -91,7 +91,9 @@ deg2rad(z::Integer) = deg2rad(float(z))
 @vectorize_1arg Real rad2deg
 @vectorize_1arg Real deg2rad
 
-log(b,x) = log(x)./log(b)
+log{T<:Number}(b::T, x::T) = log(x)/log(b)
+log(b::Number, x::Number) = log(promote(b,x)...)
+@vectorize_2arg Number log
```

Comment:
Yeah, I thought about that – it seems like overkill to test the various vectorized forms of every vectorized function since they all get vectorized the same way. But maybe we should. I feel like that might make our test time go way up though.